src/realtime_helpers.cpp

```diff
@@ -28,9 +28,7 @@
 
 #include "realtime_tools/realtime_helpers.hpp"
 
-#ifdef _WIN32
-#include <windows.h>
-#else
+#ifdef __unix__
```

Comment:
I don't know which platforms we end up targeting or want to support, but [just leaving some text here](https://web.archive.org/web/20140625123925/http://nadeausoftware.com/articles/2012/01/c_c_tip_how_use_compiler_predefined_macros_detect_operating_system#:~:text=However%2C%20compilers%20for%20OSX%2C%20iOS%2C%20and%20Darwin%20%20do%20not%20define%20__unix__.%20To%20detect%20all%20BSD%20OSes%2C%20including%20OSX%2C%20iOS%2C%20and%20Darwin%2C%20use%20an%20#if/#endif%20that%20checks%20for%20__unix__%20along%20with%20__APPLE__%20and%20__MACH__%20(see%20the%20later%20section%20on%20OSX%20and%20iOS).) to ruminate on:

> However, compilers for OSX, iOS, and Darwin do not define `__unix__`. To detect all BSD OSes, including OSX, iOS, and Darwin, use an `#if`/`#endif` that checks for `__unix__` along with `__APPLE__` and `__MACH__` (see the later section on OSX and iOS).

Reply:
```suggestion
#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
```
Make sense. I think we can use the following one as in the link you suggested  @aprotyas